apps/web/app/org/[orgSlug]/[user]/page.tsx

```diff
@@ -31,14 +31,15 @@ export const generateMetadata = async ({ params, searchParams }: PageProps) => {
         image: getOrgOrTeamAvatar(team),
       },
     };
-
+    const decodedParams = decodeParams(params);
     return {
       ...(await generateMeetingMetadata(
         meeting,
         (t) => team.name ?? t("nameless_team"),
         (t) => team.name ?? t("nameless_team"),
         false,
-        getOrgFullOrigin(currentOrgDomain ?? null)
+        getOrgFullOrigin(currentOrgDomain ?? null),
+        `/${decodedParams.user}`
```

Comment:
NIT: I think generate***Metadata seems to have too many arguments at this point 😅 (maybe a single object parameter with proper keys in the future?)